src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -787,7 +859,7 @@ static void Main()
             var tree = comp.SyntaxTrees.First();
             var model = comp.GetSemanticModel(tree);
             var def = tree.GetCompilationUnitRoot().DescendantNodes().OfType<DefaultLiteralSyntax>().Single();
-            Assert.Null(model.GetTypeInfo(def).Type);
+            Assert.Equal("System.Int32?", model.GetTypeInfo(def).Type.ToTestDisplayString());
```

Comment:
It doesn't look like constant value is tested.

---
In reply to: [102845219](https://github.com/dotnet/roslyn/pull/17205#discussion_r102845219) [](ancestors = 102845219)